test/easyconfigs/easyconfigs.py

```diff
@@ -111,7 +111,7 @@ def get_files_from_diff(diff_filter, ext):
     # first determine the 'merge base' between target branch and PR branch
     # cfr. https://git-scm.com/docs/git-merge-base
     cmd = "git merge-base %s HEAD" % target_branch
-    res = run_shell_cmd(cmd, fail_on_error=False)
+    res = run_shell_cmd(cmd, fail_on_error=False, hidden=True)
```

Comment:
these changes to the tests seem unintended for this PR, are they needed?

Reply:
Looks like this slipped in from this PR?
https://github.com/easybuilders/easybuild-easyconfigs/pull/19886

Reply:
fixed